expanderr.go

```diff
@@ -387,13 +392,30 @@ func (e *expansion) typeCheck(pkgname string, files []*ast.File) error {
 			}
 		}
 	}
+	return nil
+}
 
-	e.ce = callExprAtPath(e.path)
-	if e.ce == nil {
-		return fmt.Errorf("no ast.CallExpr found")
+type noErrTemplateFields struct {
+	Err    string // generally just "err", but not something the caller controlls
+	Return string // the entire return string
+}
+
+func noErrorReturn(fields noErrTemplateFields) (string, error) {
+	if *noErrTemplate == "" {
+		return "panic(err.Error())", nil
 	}
-	e.callee, err = signatureOf(e.info, e.ce)
-	return err
+
+	templ, err := template.New("noErr").Parse(*noErrTemplate)
```

Comment:
Can you move parsing the template to main() please? That way, we can fail early.